src/player_display.cpp

```diff
@@ -363,6 +384,16 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4" ) );
             mvwputch( w_grid_trait, i, 26, BORDER_COLOR, LINE_XOXO );
         }
 
+        //Horizontal line bionics
+        if( i <= 26 ) {
+            mvwputch( w_grid_bionics, bionics_win_size_y, i, BORDER_COLOR, LINE_OXOX );
```

Comment:
Just in case: we have `mvwhline` for horizontal lines and `mvwvline` for vertical ones.

Reply:
I'm just copying the code in player_display at this point; it clearly needs to be refactored a bunch but beyond the scope of this PR.